Cargo.toml

```diff
@@ -16,7 +16,7 @@ build = "build.rs"
 colored = "1.5"
 difference = "1.0"
 error-chain = "0.11"
-rustc-serialize = "0.3"
+json = "0.11"
```

Comment:
I'd usually prefer to use serde_json everywhere, but it's fine to go with this crate here as it does what we need it to do and also has no dependencies.

Reply:
I went with it for that and because I thought `serde_json` was implemented in terms of it which I just found out it isn't.

Reply:
I've gone ahead and switched it to `serde_json` (already shows up in the dependency tree anyways).